backend/src/main/resources/.gitignore

```diff
@@ -0,0 +1 @@
+keys/
```

Comment:
I assume this is to ignore the key.json file right?

Reply:
yep!